run.sh

```diff
@@ -14,5 +14,5 @@ migrate)
     bundle exec rake db:schema:load
     ;;
 esac
-bundle exec import:all assets:precompile
+RAILS_ENV=production bundle exec import:all assets:precompile
```

Comment:
With that said, if you do want to run tests in Jenkins, then yes, you'll need a `test` environment. However, in that case, you don't need to build assets at all. So what I don't understand is why you'd ever need to run the asset compilation in the `production` `RAILS_ENV` – if you're compiling assets, you should already be in `production`.